docs/nv2/README.md

```diff
@@ -72,6 +72,7 @@ OPTIONS:
    --username value, -u value   username for generic remote access
    --password value, -p value   password for generic remote access
    --insecure                   enable insecure remote access (default: false)
+   --media-type value           specify the media type of the manifest read from file or stdin (default: "application/vnd.docker.distribution.manifest.v2+json")
```

Comment:
Curious why we need to specify the `mediaType`. Does it matter when evaluating an nv2 signature?

Reply:
When the manifest comes from a file or `stdin`, the `nv2` does not know the media type of the manifest. It can be a docker manfiest or oci manifest or even a list. Therefore, you need to specify the media type. If you get the manifest by `docker://` or `oci:`, the `nv2` will set the media type for you.

Since the media type is there with size and digest, it is also verified when evaluating a nv2 signature.

Reply:
I'm not sure I follow the flow, but can we make it `--mediaType`?